app/Http/Controllers/Staff/UploaderController.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * NOTICE OF LICENSE.
+ *
+ * UNIT3D Community Edition is open-sourced software licensed under the GNU Affero General Public License v3.0
+ * The details is bundled with this project in the file LICENSE.txt.
+ *
+ * @project    UNIT3D Community Edition
+ *
+ * @author     HDVinnie <hdinnovations@protonmail.com>
+ * @license    https://www.gnu.org/licenses/agpl-3.0.en.html/ GNU Affero General Public License v3.0
+ */
+
+namespace App\Http\Controllers\Staff;
+
+use App\Http\Controllers\Controller;
+use App\Models\Group;
+use App\Models\User;
+
+class UploaderController extends Controller
+{
+    /**
+     * Display Stats.
+     */
+    public function index(): \Illuminate\Contracts\View\Factory|\Illuminate\View\View
+    {
+        $uploaderGroup = cache()->rememberForever('uploader_group', fn () => Group::where('is_uploader', 1)->pluck('id'));
+
+        return view('Staff.uploader.index', [
+            'uploaders' => User::with(['group'])
+                ->where('group_id', '=', $uploaderGroup)
+                // Count total uploads for current user
+                ->withCount(['torrents as total_uploads' => fn ($query) => $query
+                    ->whereColumn('user_id', 'users.id')
```

Comment:
The query builder should already be adding this constraint on its own. Can you double check if it's actually needed? I believe `->withCount('torrents as total_uploads')` should work fine on its own

Reply:
Removed, as not needed.